### PR TITLE
#184 - Fix getting file from archive by name

### LINKED ISSUE #184: Exact match is used to obtain files from .tgz files. Fuzzy match may fail to obtain correct files.
for example, multiple files in the compressed package have the same name but different paths.
at:
com.artipie.npm.TgzArchive#file

### PR DESCRIPTION
**Pull request by genryxy**
Closes #184 
Fixed getting file from archive by name

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -301,6 +301,15 @@ SOFTWARE.
           </execution>
         </executions>
       </plugin>
+      <plugin>
+        <groupId>com.qulice</groupId>
+        <artifactId>qulice-maven-plugin</artifactId>
```
Comment (g4s8) [inline pom.xml:306]: @genryxy this plugin is enabled in `qulice` profile in parent pom. In this PR you're enabling it by default (outside of profile) with overridden configuration. If you want to extend plugin configuration defined in parent pom, the correct way is to specify it in `pluginManagement` section, e.g. like here: https://github.com/artipie/artipie/blob/master/pom.xml#L383
```

**Review by g4s8 — CHANGES_REQUESTED** [review]
@genryxy please check my comment

**Comment by genryxy** [comment]
@g4s8 thanks, added `pluginManagement` section

**Comment by genryxy** [comment]
@g4s8 could you please check/merge?

**Review by g4s8 — APPROVED** [review]
@genryxy thanks

**Comment by 0crat** [comment]
Job `gh:artipie/npm-adapter#191` is not assigned, can't get performer

**Pull request merged**